exasol_script_languages_container_ci_setup/templates/build_buildspec.yaml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    BUILD_DOCKER_REPOSITORY: "exadockerci4/script-languages-build-cache"
+    RELEASE_DOCKER_REPOSITORY: "exasol/script-language-container"
+  secrets-manager:
+    DOCKER_USER: "Dockerhub:User"
+    DOCKER_PASSWORD: "Dockerhub:AccessToken"
+phases:
+
+  install:
+    runtime-versions:
+      python: 3.8
+    commands:
+      - git submodule update --init --recursive
+      - curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+      - export PATH=$PATH:$HOME/.poetry/bin
+      - poetry env use $(command -v "python3.8")
+      - poetry install
+
+  pre_build:
+      commands:
+        - echo CODEBUILD_SOURCE_VERSION is $CODEBUILD_SOURCE_VERSION #supposed to be the SHA
+        - echo CODEBUILD_WEBHOOK_HEAD_REF is $CODEBUILD_WEBHOOK_HEAD_REF #supposed to the branch name
+        - echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USER" --password-stdin
+  build:
+      commands:
+        - poetry run python3 -m exasol_script_languages_container_ci.main run-ci --flavor $FLAVOR --branch-name "$CODEBUILD_WEBHOOK_HEAD_REF" --docker-user "$DOCKER_USER" --docker-password "$DOCKER_PASSWORD" --docker-build-repository "$BUILD_DOCKER_REPOSITORY" --docker-release-repository "$RELEASE_DOCKER_REPOSITORY" --commit-sha "$CODEBUILD_SOURCE_VERSION" {{ config_file_parameter }}
```

Comment:
is this docker user and docker password safe? Can use environment variables instead?

Reply:
AWS automatically replaces all strings coming from the secret-manager with "***", in all logs. So I think using the parameter here should be safe.